.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -15,15 +19,14 @@ about: Report an issue or question while using nni instance (deployment).
 - Is conda/virtualenv/venv used?:
 - Is running in Docker?:
 
+
+
 **Log message**:
- - nnimanager.log: 
- - dispatcher.log:
- - nnictl stdout and stderr:
+ - **nnimanager.log:**
+ - **dispatcher.log:**
+ - **nnictl stdout and stderr:**
  
 <!-- Where can you find the log files: [log](https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/HowToDebug.md#experiment-root-director), [stdout/stderr](https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/Nnictl.md#nnictl%20log%20stdout) -->
```

Comment:
I think it's meaningless to use markdown in comment.
Suggest change to this format:
```
<!--
Where can you find the log files:
LOG: https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/HowToDebug.md#experiment-root-director
STDOUT/STDERR: https://github.com/microsoft/nni/blob/master/docs/en_US/Tutorial/Nnictl.md#nnictl%20log%20stdout
-->
```

Reply:
yes